panel/io/jupyter_server_extension.py

```diff
@@ -112,6 +112,8 @@ def get_server_root_dir(settings):
 import pathlib
 import sys
 
+if os.environ.get('BOKEH_RESOURCES') == 'inline':
```

Comment:
I don't know if the if statement is needed.

Reply:
Yes, let's drop it and just force it.